### ISSUE TITLE
Depricated method isAlive

### ISSUE THREAD

**Issue by titiviking** [araines/homeassistant-thermoplus-ble#11]
Home assistant is returning errors for Thermoplus_BLE integration:

```
/config/custom_components/thermoplus_ble/ble.py:53: DeprecationWarning: isAlive() is deprecated, use is_alive() instead


  if self.thread.isAlive():


/config/custom_components/thermoplus_ble/ble.py:55: DeprecationWarning: isAlive() is deprecated, use is_alive() instead


  if self.thread.isAlive():
```

**Comment by araines** [comment]
Thanks for the report, I'll take a look in to this soon

**Comment by araines** [comment]
Fixed in v0.2.2
